.travis.yml

```diff
@@ -10,6 +10,7 @@ env:
         - PHPUNIT_FLAGS="-v"
         - SYMFONY_PHPUNIT_DIR="$HOME/symfony-bridge/.phpunit"
         - SYMFONY_REQUIRE='>=2.8'
+        - SYMFONY_DEPRECATIONS_HELPER="weak"
```

Comment:
This has been changed because Symfony deprecations on the router component configuration.
There should probably be better ways to deal with it, maybe with Symfony version's specific configurations but I'm not sure it makes much sense.
I'm open to better ideas!